test/integration/scheduler/preemption_test.go

```diff
@@ -731,119 +730,6 @@ func TestPreemptionStarvation(t *testing.T) {
 	}
 }
 
-// TestPreemptionRaces tests that other scheduling events and operations do not
-// race with the preemption process.
-func TestPreemptionRaces(t *testing.T) {
```

Comment:
Per the [slack discussion](https://kubernetes.slack.com/archives/C09TP78DV/p1590677102242500), let's add it back, and fix the flake.